validator_client/src/cli.rs

```diff
@@ -1,478 +1,466 @@
-                .long("web3-signer-max-idle-connections")
-                .value_name("COUNT")
-                .help("Maximum number of idle connections to maintain per web3signer host. Default \
-                       is unlimited.")
-                .action(ArgAction::Set)
-                .display_order(0)
-        )
+                and never provide an untrusted URL.",
+        display_order = 0
+    )]
+    pub monitoring_endpoint: Option<String>,
+
+    #[clap(
+        long,
+        value_name = "SECONDS",
+        requires = "monitoring_endpoint",
+        default_value_t = 60,
+        help = "Defines how many seconds to wait between each message sent to \
+                the monitoring-endpoint.",
+        display_order = 0
+    )]
+    pub monitoring_endpoint_period: u64,
+
+    #[clap(
+        long,
+        value_name = "ENABLE_DOPPELGANGER_PROTECTION",
+        help = "If this flag is set, Lighthouse will delay startup for three epochs and \
+                monitor for messages on the network by any of the validators managed by this \
+                client. This will result in three (possibly four) epochs worth of missed \
+                attestations. If an attestation is detected during this period, it means it is \
+                very likely that you are running a second validator client with the same keys. \
+                This validator client will immediately shutdown if this is detected in order \
+                to avoid potentially committing a slashable offense. Use this flag in order to \
+                ENABLE this functionality, without this flag Lighthouse will begin attesting \
+                immediately.",
+        display_order = 0,
+        help_heading = FLAG_HEADER
+    )]
+    pub enable_doppelganger_protection: bool,
+
+    #[clap(
+        long,
+        alias = "private-tx-proposals",
+        help = "If this flag is set, Lighthouse will query the Beacon Node for only block \
+                headers during proposals and will sign over headers. Useful for outsourcing \
+                execution payload construction during proposals.",
+        display_order = 0,
+        help_heading = FLAG_HEADER
+    )]
+    pub builder_proposals: bool,
+
+    #[clap(
+        long,
+        value_name = "UNIX-TIMESTAMP",
+        help = "This flag takes a unix timestamp value that will be used to override the \
+                timestamp used in the builder api registration.",
+        display_order = 0
+    )]
+    pub builder_registration_timestamp_override: Option<u64>,
+
+    #[clap(
+        long,
+        value_name = "INTEGER",
+        default_value_t = 30_000_000,
+        help = "The gas limit to be used in all builder proposals for all validators managed \
+                by this validator client. Note this will not necessarily be used if the gas limit \
+                set here moves too far from the previous block's gas limit.",
+        display_order = 0
+    )]
```

Comment:
`.requires("builder-proposals")` is no longer here, is it intentional?

Reply:
not intentional, ive re-added it